auto_update.go

```diff
@@ -125,16 +131,19 @@ func getPlatformZipURL(version string) string {
 }
 
 func getLatestVersion() (string, error) {
-	resp, err := http.Get("https://coveo-bootstrap-us-east-1.s3.amazonaws.com/tgf_version.txt")
+	resp, err := http.Get("https://api.github.com/repositories/91362181/releases/latest")
```

Comment:
`https://api.github.com/repos/coveooss/tgf/releases/latest` is nicer than an internal Github ID. I verified and `coveo/tgf` still works so it is no less retrocompatible